src/components/Sokeresultat.tsx

```diff
@@ -57,12 +58,18 @@ const Sokeresultat = ({ allEvents }: SokeresultatProps) => {
     setMarkertePersoner([]);
   }, [tabType]);
 
+  const selectedHendelsetypeFilter =
```

Comment:
La på denne for å unngå at man gjør filtrering på ufordelte brukere i Min oversikt i tilfeller hvor man har filtrert på dette i Enhetens oversikt og så gått til Min oversikt (via menyen i dekoratøren).